sap_hana/tests/test_config.py

```diff
@@ -0,0 +1,16 @@
+# (C) Datadog, Inc. 2022-present
+# All rights reserved
+# Licensed under a 3-clause BSD style license (see LICENSE)
+
+from datadog_checks.sap_hana import SapHanaCheck
+
+
+def test_default_tenant_databases(instance):
+    check = SapHanaCheck('sap_hana', {}, [instance])
+    assert check._tenant_databases_schema == "SYS_DATABASES"
+
+
```

Comment:
If it's easy to add, could you include a test where we configure an invalid `schema`, and then the check run fails?

Reply:
The problem is that the check does not fail if one of the query fails:

Reply:
Test added :) https://github.com/DataDog/integrations-core/pull/12492/commits/06ecfd7115d7b1384ada7834222608d500284fb3

Reply:
Looks great!